src/main/java/com/github/webicitybrowser/spec/fetch/FetchConsumeBodyAction.java

```diff
@@ -1,7 +1,9 @@
 package com.github.webicitybrowser.spec.fetch;
 
 public interface FetchConsumeBodyAction {
-	
+
+
 	void execute(FetchResponse response, boolean success, byte[] body);
```

Comment:
I put a list of styling guidelines on Discord, based off of any stylistic things I've seen
I'll make a styling document at some point